Dima/text/check_if_palindrome/check_if_palindrome.py

```diff
@@ -0,0 +1,18 @@
+def check_if_palindrome(palindrome):
+    try:
+        palindrome = palindrome.replace(" ", "")
```

Comment:
There is python function for this action
https://www.journaldev.com/23625/python-trim-string-rstrip-lstrip-strip
please use it

Reply:
Yes, i know, i`ll use it to remove some special symbols, but i used the replace(" ", "") function to add possibility to check palindrome phrases, not only words in string. I added new test for it.